docs/src/operations/requirements.md

```diff
@@ -18,50 +18,15 @@ transaction for each block the validator agrees with, which can cost up to
 
 The hardware recommendations below are provided as a guide.  Operators are encouraged to do their own performance testing.
 
-- CPU
-  - 12 cores / 24 threads, or more
-  - 2.8GHz base clock speed, or faster
-  - SHA extensions instruction support
-  - AMD Gen 3 or newer
-  - Intel Ice Lake or newer
-  - Higher clock speed is preferable over more cores
-  - AVX2 instruction support (to use official release binaries, self-compile
-    otherwise)
-  - Support for AVX512f is helpful
-- RAM
-  - 256GB or more
-  - Error Correction Code (ECC) memory is suggested
-  - Motherboard with 512GB capacity suggested
-- Disk
-  - PCIe Gen3 x4 NVME SSD, or better
-  - Accounts: 500GB, or larger. High TBW (Total Bytes Written)
-  - Ledger: 1TB or larger. High TBW suggested
-  - Snapshots: 250GB or larger. High TBW suggested
-  - OS: (Optional) 500GB, or larger. SATA OK
-  - The OS may be installed on the ledger disk, though testing has shown better
-    performance with the ledger on its own disk
-  - Accounts and ledger _can_ be stored on the same disk, however due to high
-    IOPS, this is not recommended
-  - The Samsung 970 and 980 Pro series SSDs are popular with the validator community
-- GPUs
-  - Not necessary at this time
-  - Operators in the validator community do not use GPUs currently
-
-### RPC Node Recommendations
-
-The [hardware recommendations](#hardware-recommendations) above should be considered
-bare minimums if the validator is intended to be employed as an RPC node. To provide
-full functionality and improved reliability, the following adjustments should be
-made.
-
-- CPU
-  - 16 cores / 32 threads, or more
-- RAM
-  - **1.17** 512 GB or more if an `account-index` is used, 1TB+ for all three [account indexes](https://docs.solanalabs.com/operations/setup-an-rpc-node#account-indexing)
-  - **1.18 or newer** 512 GB or more for all three indexes
```

Comment:
For the sake of papertrail - we dropped the v1.17 / v1.18 line here given that MNB is currently running v1.18